srcread.py

```diff
@@ -0,0 +1,4 @@
+def findparameterByType(linesList, parameterType):
```

Comment:
Instead of just writing a function, make it with class, which initializes with a file. Whenever we want to inject a value of a variable from any source file, we will just create an object of type let's say `SRCRead`, with a source file being its constructor argument. 
In our case, we will make an object with `source.txt` being the mentioned above source file.